code/__HELPERS/bodyparts.dm

```diff
@@ -1 +1,7 @@
 #define IS_ORGANIC_LIMB(limb) (limb.bodytype & BODYTYPE_ORGANIC)
+
+/mob/living/carbon/proc/del_and_replace_bodypart(obj/item/bodypart/new_limb, special)
```

Comment:
1. Document this
2. Put this in the mob folder, __HELPERS should be kept somewhat slim/global

Reply:
Document where

Reply:
A triple slashed comment makes a doc comment

```
/// Doc comment goes here
/thing/proc/action()
```

code/modules/mob/living/carbon, and make like carbon_bodyparts.dm

Reply:
why not put it in dismember code

Reply:
Sure just not in a helper

Reply:
Done